### PR TITLE
Add entity diagnostic logs

### PR DESCRIPTION
**Pull request by xirc**
Add entity (typed) diagnostic logs:
* Containing the current entity state (`Inactive`, `Ready`, `Recovering`, `WaitForReplication`)
* Logging as TRACE level

### CODE REVIEW

File: src/main/scala/lerna/akka/entityreplication/typed/internal/behavior/Recovering.scala [hunk 1]
```diff
@@ -59,10 +76,30 @@ private[entityreplication] class Recovering[Command, Event, State](
...
                 case None           => LogEntryIndex.initial()
               }
+              if (context.log.isTraceEnabled) {
+                context.log.trace(
+                  "[{}] Sending FetchEntityEvents: entityId=[{}], fromIndex=[{}], toIndex=[{}], replyTo=[{}], to=[{}]",
+                  stateName,
+                  setup.replicationId.entityId.raw,
+                  snapshotIndex.next(),
+                  recoveryIndex,
+                  fetchEntityEventsResponseMapper(command.entitySnapshot),
```
Comment (negokaz) [inline src/main/scala/lerna/akka/entityreplication/typed/internal/behavior/Recovering.scala:99]: `fetchEntityEventsResponseMapper` creates a new `messageAdapter`.
How about we call this only once?

Or we might be able to exclude this from the message.

**Review by negokaz — COMMENTED** [review]
@xirc 

Great! I left a comment.

File: src/main/scala/lerna/akka/entityreplication/typed/internal/behavior/Recovering.scala (hunk 1 above)
Comment (xirc) [reply src/main/scala/lerna/akka/entityreplication/typed/internal/behavior/Recovering.scala]: @negokaz I've changed the code such that fetchEntityEventsResponseMapper is called only once.

**Review by negokaz — APPROVED** [review]
Great! LGTM 👍

**Pull request merged**